README.md

```diff
@@ -260,8 +265,466 @@ All project configuration properties are located in **_config.properties** file.
 		<td>{must_override}</td>
 		<td>pwd123</td>
 	</tr>
-</table>
-
+	<tr>
+		<td>env</td>
+		<td>Environment specific configuration. More about this [feature](#environment-specific-configuration)</td>
+		<td>n/a</td>
+		<td>STAG, PROD, DEMO</td>
+	</tr>
+	<tr>
+		<td>env_arg_resolver</td>
+		<td>This parametr is optional, if it isn't set default value will be used. In most cases <b>default value is enough</b></td>
+		<td>com.qaprosoft.carina.core.foundation.utils.DefaultEnvArgResolver</td>
+		<td>java class </td>
+	</tr>
+		<tr>
+		<td>platform</td>
+		<td>Platform version for Selenium Grid</td>
+		<td>*</td>
+		<td>ANDROID,IOS,WINDOWS,MAC,LINUX</td>
+	</tr>
+		<tr>
+		<td>browser_version</td>
+		<td>The browser version, or the empty string if unknown for Selenium Grid</td>
+		<td>n/a</td>
+		<td>"8.0", "52.1"</td>
+	</tr>
+		<tr>
+		<td>driver_mode</td>
+		<td>Rule for defining WebDriver lifecycle.</td>
+		<td>method_mode</td>
+		<td>method_mode / class_mode / suite_mode</td>
+	</tr>
+		<tr>
+		<td>driver_mode</td>
+		<td>Rule for defining WebDriver lifecycle.</td>
+		<td>method_mode</td>
+		<td>method_mode / class_mode / suite_mode</td>
+	</tr>
+		<tr>
+		<td>driver_event_listeners</td>
+		<td>Comma-separated list of listeners. Listeners provide more logs from WebDriver and have to be instances of WebDriverEventListener</td>
+		<td>n/a</td>
+		<td>com.someCompane.core.EventListener1,com.someCompane.core.EventListener2</td>
+	</tr>
+		<tr>
+		<td>max_driver_count</td>
+		<td>Max number of drivers per thread</td>
+		<td>3</td>
+		<td>Integer</td>
+	</tr>
+		<tr>
+		<td>custom_capabilities</td>
+		<td>Name of propertie file with custom capabilities(key-value)</td>
+		<td>n/a</td>
+		<td>custom.propertie</td>
+	</tr>
+		<tr>
+		<td>proxy_host</td>
+		<td>The hostname of the server</td>
+		<td>n/a</td>
+		<td>host.example.com</td>
+	</tr>
+		<tr>
+		<td>proxy_port</td>
+		<td>The port number</td>
+		<td>n/a</td>
+		<td>80</td>
+	</tr>
+		<tr>
+		<td>proxy_protocols</td>
+		<td>Comma-separated list of internet protocols used to carry connection information from the source requesting the connection to the destination for which the connection was requested.</td>
+		<td>http,https,ftp</td>
+		<td>http, https, ftp, socks</td>
+	</tr>
+		<tr>
+		<td>browsermob_proxy</td>
+		<td>Boolean parameter which enable or disable set up of mobile proxy</td>
+		<td>false</td>
+		<td>true, false</td>
+	</tr>
+		<tr>
+		<td>browsermob_port</td>
+		<td>The port number for mobile browser (make sense only for local debugging)</td>
+		<td>0</td>
+		<td>Integer</td>
+	</tr>
+		<tr>
+		<td>proxy_set_to_system</td>
+		<td>Boolean parameter which enable or disable set up of proxy</td>
```

Comment:
Boolean parameter which enable or disable set up of proxy
->
Boolean parameter which enable or disable set up of proxy for test java process. When disabled only drivers use proxy.